daily_milestone.go

```diff
@@ -222,8 +245,8 @@ func main() {
 	var token, baseURL, namespace, project, timeInterval string
 	var advance int
 	// Command Line Parsing Starts
-	flag.StringVar(&token, "Token", "jGWPwqQUuf37b", "Gitlab api key/token")
-	flag.StringVar(&timeInterval, "TimeInterval", "daily", "Milestone interval for daily, weekly or monthly")
+	flag.StringVar(&token, "Token", "jGWPwqQUuf37b", "Gitlab api key/token.")
+	flag.StringVar(&timeInterval, "TimeInterval", "daily", "to set milestone to daily, weekly or monthly.")
```

Comment:
You readded some stuff here, which was already fixed. You might wanna change that back. Commit and rebase and push again.

Reply:
This still needs fixing.